clevertap-android-sdk/src/main/java/com/clevertap/android/sdk/Validator.java

```diff
@@ -305,6 +316,33 @@ ValidationResult isRestrictedEventName(String name) {
         return error;
     }
 
+    /**
+     * Checks whether the specified event name has been discarded from Dashboard. If it is,
+     * then create a pending error, and abort.
+     *
+     * @param name The event name
+     * @return Boolean indication whether the event name has been discarded from Dashboard
+     */
+    ValidationResult isEventDiscarded(String name) {
+        ValidationResult error = new ValidationResult();
+        if (name == null) {
+            error.setErrorCode(510);
+            error.setErrorDesc("Event Name is null");
+            return error;
+        }
+        if(getDiscardedEvents() != null) {
+            for (String x : getDiscardedEvents())
+                if (name.equalsIgnoreCase(x)) {
+                    // The event name is discarded
+                    error.setErrorCode(513);
+                    error.setErrorDesc(name + " is a discarded event name. Last event aborted.");
+                    Logger.v(name + " is a discarded event name. Last event aborted.");
```

Comment:
@kumaratulismu change the error description and logging to this ->
error.setErrorDesc(name + " is a discarded event name as per CleverTap. Dropping event at SDK level. Check discarded events in Dashboard settings.");
                    Logger.v(name + " s a discarded event name as per CleverTap. Dropping event at SDK level. Check discarded events in Dashboard settings.");